lib/wkt.js

```diff
@@ -111,7 +111,11 @@ function cleanWKT(wkt) {
       wkt.units = 'meter';
     }
     if (wkt.UNIT.convert) {
-      wkt.to_meter = parseFloat(wkt.UNIT.convert, 10);
+      if (wkt.type === 'GEOGCS') {
+        wkt.to_meter = parseFloat(wkt.UNIT.convert, 10)*6370997;
```

Comment:
if `wkt.type === 'GEOGCS'` but `DATUM` is undefined, what should the behaviour be?
* use 6370997 ("Normal" sphere)
* throw an error
* do not set to_meter